src/conduits/RwaInputConduit3.sol

```diff
@@ -0,0 +1,310 @@
+// SPDX-FileCopyrightText: © 2020-2021 Lev Livnev <lev@liv.nev.org.uk>
```

Comment:
Only creation year would need to be specified following Dai Foundation [guidelines](https://github.com/daifoundation/docs/blob/master/COPYRIGHT-GUIDANCE.md).

Reply:
done